gapic/src/main/com/google/gapid/image/ArrayImage.java

```diff
@@ -541,6 +546,65 @@ public boolean isDark() {
     }
   }
 
+  /**
+   * An {@link ArrayImage} that represents an 8bit count image.
+   */
+  public static class Count8Image extends Luminance8Image {
```

Comment:
Don't extend `Luminance8Image`, you are overriding practically all methods anyways.

Reply:
Done